Common/src/CConfig.cpp

```diff
@@ -4730,7 +4730,8 @@ void CConfig::SetPostprocessing(SU2_COMPONENT val_software, unsigned short val_i
 
   if (Kind_Solver == INC_EULER || Kind_Solver == INC_NAVIER_STOKES || Kind_Solver == INC_RANS) {
     if ((Kind_DensityModel == INC_DENSITYMODEL::CONSTANT) || (Kind_DensityModel == INC_DENSITYMODEL::BOUSSINESQ))
-      Kind_FluidModel = CONSTANT_DENSITY;
+      if (Kind_FluidModel != CONSTANT_DENSITY)
+        SU2_MPI::Error("Incompressible problems with DENSITY_MODEL = CONSTANT or DENSITY_MODEL = BOUSSINESQ must use FLUID_MODEL = CONSTANT_DENSITY.", CURRENT_FUNCTION);
```

Comment:
Problem here was that the default `FLUID_MODEL `is `STANDARD_AIR `and the failing reg cases relied upon this code bit  changing the not-set `FLUID_MODEL `to `CONSTANT_DENSITY`. I prefer being more strict in the config (erroring instead of silently changing), because this would also silently change the  `FLUID_MODEL` back to `CONSTANT_DENSITY` although the user provided a different  `FLUID_MODEL`. This, to some extent, also forces one to learn more about what combinations are reasonable and why others aren't. 

This wont be handeled here but maybe somewhere else